RPIPs/RPIP-bounty_standards.md

```diff
@@ -0,0 +1,101 @@
+author: LongForWisdom (@LongForWisdom)
+discussions-to: <URL>
+status: Draft
+type: Meta
+created: 2024-01-04
+requires: 26
+---
+
+## Abstract
+
+TODO Write
+
+## Motivation
+
+TODO Write
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119. TODO Remove
+
+### Bounty Lifecycle
+![image](../assets/rpip-bounty_standards/bounty-lifecycle.png)
+
+#### Statuses
+**Draft - Proposed** - Bounty is described and proposed to the GMC for adoption. Bounty is not yet eligible for adoption.  
+**Draft - Defined** - Bounty is well defined and ready for completion by a bounty hunter with the relevant skillsets if adopted by the GMC. Bounty is now eligible for adoption.  
+**Draft - Rejected** - Bounty was rejected by the GMC at this time. Bounty is not eligible for completion.  
+**Open** - Bounty is adopted by the GMC for completion and listed on relevant frontends. Bounty is now eligible for completion.  
+**Closed - Completed** - A Bounty has been fully completed and paid out by the GMC. Bounty is no longer eligible for completion.  
+**Closed - Expired** - Bounty has met its expiration date. Bounty is no longer eligible for completion.  
+**Closed - Withdrawn** - GMC has withdrawn the bounty. Bounty is no longer eligible for completion.  
+
+### Bounty Incentive Levers
+Bounty Incentives Levers are OPTIONAL incentives that the pDAO empowers the GMC to use to incentivise the generation and successful completion of bounty proposals.
+
+#### Lever Incentives
+* The incentive attached to each lever MAY be set by the GMC. 
+* Levers MAY have an absolute (fixed amount) component and/or a relative (percentage of total bounty payout) component. 
+* The payout of incentives for a given bounty MUST NOT reduce the bounty completion payout. 
+* When incentive levers are modified by the GMC, incentives SHOULD be paid out in such a way that ambiguity favours incentive recipients.
+* The total cost of lever incentives is not fixed. The GMC SHOULD set lever incentives with caution when funds are limited. 
+
+#### Payout
+* Bounty incentives SHOULD be paid out of the GMC budget.
+* Incentive payouts SHOULD take place within the GMC's regular payout schedule.
+* Incentive payouts MUST be denominated in dollars, and MUST be paid out in one of the GMC's accepted payout token options selected by the recipient.
+
+#### Lever A: Proposal Incentive
+The proposal incentive lever aims to influence the number of bounty proposals that are recieved by the GMC.
+* This incentive MUST be paid to the proposal author for a bounty proposal that is adopted by the GMC. 
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on proposal quality and completeness.
+* Bounty definitions do not need to be included for a proposer to recieve this incentive.
+
+#### Lever B: Definition Incentive
```

Comment:
Can you elaborate on the need for a Definition Incentive Lever and a Proposal Incentive Lever? The proposal should contain enough definition to define the desired outcome; otherwise, it's a naff proposal. It would also protect against spamming proposals if people need to work more to get the incentive.

Reply:
I might be missing the point. Perhaps "Definition" needs to be defined.

Reply:
I'm split on this one as well, seems like a solid idea as there is often extra labor the proposer feels everything is well defined but people less connected to the genre/process of the bounty might be lost.

Main issue I have is I think it will add a lot of labor to the GMC to be going back and forth about definitions, especially with the bias for 1 approved submission. Would it work a bit better to have a definitions "pot" that the GMC MAY tip out of for helpful definitions submissions? That way the GMC doesn't have to have a long back and forth, could just say "hey we would like to see this section defined a bit better for the final state, so we awarded 75% of the definition pot and plan to pay the rest out when XYZ is addressed.

Reply:
Whole point is to split the high-effort definition from the low-effort proposal. Want to lower the barrier for entry for proposals, but still have a baseline of quality for bounty hunters due to the requirement for definition. 

Added a definitions section based on your confusion.